src/components/queryBuilder/LogLevelField.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { SelectableValue } from '@grafana/data';
+import { InlineFormLabel, Select } from '@grafana/ui';
+import { selectors } from './../../selectors';
+import { FullField } from 'types';
+
+interface LogLevelEditorProps {
+  fieldsList: FullField[];
+  onLogLevelFieldChange: (logLevelField: string, logLevelFieldType: string) => void;
+}
+
+export const LogLevelFieldEditor = (props: LogLevelEditorProps) => {
+  const { label, tooltip } = selectors.components.QueryEditor.QueryBuilder.LOG_LEVEL_FIELD;
+  // TODO: filter by strings, enums?
```

Comment:
I guess here we might see
* String
* FixedString
* Enum
* LowCardinality
* Nullable of all above

Reply:
Will be done as a follow-up